src/Microsoft.AspNetCore.Testing/Microsoft.AspNetCore.Testing.csproj

```diff
@@ -16,4 +16,9 @@
     <PackageReference Include="xunit" Version="$(XunitVersion)" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetFrameworkIdentifier)'=='.NETFramework'">
```

Comment:
When we don't reference `System.Net.Http` and `System.Runtime.InteropServices.RuntimeInformation` it can't find `HttpContent` and `OSPlatforms` (among other things). Where should we get those members from instead?